MAINTENANCE.md

```diff
@@ -1,13 +1,23 @@
 This document explains how to perform the project's maintenance tasks.
 
+### Repository Configuration
+
+Settings can be changed here by means of a PR - depending on what it is a conversation about pro's and cons might be needed.
+
+https://github.com/rust-lang/team/blob/master/repos/rust-lang/flate2-rs.toml
+
+You can send a PR to add `required-approvals = 0` to the configuration file above. See [this configuration](https://github.com/rust-lang/team/blob/master/repos/rust-lang/libc.toml) 
+for an example.
+
+
 ### Creating a new release
 
 #### Artifacts
 
 * a tag of the version number
 * a new [crate version](https://crates.io/crates/flate2/versions)
 
-#### Process
+#### Process: Plain `cargo publish`
```

Comment:
I would like an introductory paragraph to describe why there are two release processes, and which one to use by default.
```suggestion
#### Process
We are moving to using `cargo smart-release` for releases, but it still needs some polishing. [Some criteria to choose which one to use.]
##### `cargo publish`
...
##### `cargo smart-release`
...
```

Reply:
Actually, this could be up to discussion if having two processes even makes sense.
One day I just 'felt' very strongly about wanting to use `cargo smart-release` but didn't think of the incompatibility regarding the tags, and the other problems that might arise from a PR-based workflow which `cargo smart-release` isn't designed around (so `gh` invocations might fail).

Even though Changelogs are nice, in the end, it might be easier to remove it and call it a failed experiment - I'd actually vote for that.

Reply:
Reverting the changes from the Process header down would be clearer. For changelogs, is it possible to add `cargo changelog` to the `cargo publish` workflow?

Reply:
That’s definitely possible.